apps/pano/src/routes/send.tsx

```diff
@@ -87,7 +91,10 @@ const Send = () => {
             name="url"
             size="2"
             onPaste={(e) => {
-              onPaste(e.clipboardData.getData("text"));
+              fetchMetaTags(e.clipboardData.getData("text"));
+            }}
+            onBlur={(e) => {
+              fetchMetaTags(e.target.value);
```

Comment:
`onBlur` is still late for doing this. User needs to know that AFTER they leave the input, ONLY THEN it will be fetched, which is wrong IMO.

The correct way to fix this should be using a `debounced` callback for `onChange` event handler.